### ISSUE TITLE
RabbitMQ Support

### ISSUE THREAD

**Issue by jpeyper** [zendesk/maxwell#311]
Hi :)

We are currently using the Kafka producer, most of our downstream use however is taking off this stream and posting on RabbitMQ (AMQP) queue for a worker service to do something with.

Ideally, removing the Kafka infrastructure from our system would be good, and if Maxwell was able to directly post to a queue rather than a stream, I think it might be useful for more than just us.

How likely is it to have support added?

**Comment by osheroff** [comment]
@jpeyper Zendesk doesn't use AMQP so it's pretty unlikely to come from my organization; I take some liberties with my time to add features to Maxwell that the community requests (but I might not need myself), with the goal of spreading adoption and getting features contributed back.  That's the promise of open-source, and when it works it's glorious.  

But what open-source isn't is a thing where you can just ask for features from the community (who are often developers paid to do other things) and hope that someone else puts the time in. 

In other words @jpeyper: put your code where your mouth is.  Give me some of your developer's time (like I give you mine).  Patches welcome.

**Comment by jpeyper** [comment]
@osheroff I suspected this was the case.

I'll try to get up to speed on the Maxwell code base and provide support myself :)

**Comment by Hiteshm01** [comment]
Hi @jpeyper @osheroff 

I have attempted to add rabbitmq support to maxwell in this PR https://github.com/zendesk/maxwell/pull/525. Please review/suggest.
